.cirrus.yml

```diff
@@ -0,0 +1,95 @@
+env:
+  # Determine what we need here
+
+eks_container: &BUILD_TEMPLATE
+  dockerfile: .cirrus/Dockerfile
+  builder_subnet_id: ${CIRRUS_AWS_SUBNET}
+  builder_role: cirrus-builder
+  builder_image: docker-builder-v*
+  builder_instance_type: t2.small
+  cluster_name: ${CIRRUS_CLUSTER_NAME}
+  region: eu-central-1
+  namespace: default
+  docker_arguments:
+    CIRRUS_AWS_ACCOUNT: ${CIRRUS_AWS_ACCOUNT}
+
+yarn_cache_template: &YARN_CACHE_TEMPLATE
+  yarn_cache:
+    folders:
+      - '.yarn'
+    fingerprint_script: cat yarn.lock
+
+eslint_report_cache_template: &ESLINT_REPORT_CACHE_TEMPLATE
+  eslint_report_cache:
+    folder: 'build-reports/eslint-report'
+    fingerprint_script: echo $CIRRUS_BUILD_ID
+
+jest_coverage_cache_template: &JEST_COVERAGE_CACHE_TEMPLATE
+  jest_coverage_cache:
+    folder: 'build-reports/coverage'
+    fingerprint_script: echo $CIRRUS_BUILD_ID
+
+build_dist_cache_template: &BUILD_DIST_CACHE_TEMPLATE
+  build_dist_cache:
+    folder: 'dist/**'
+    fingerprint_script: echo $CIRRUS_BUILD_ID
+
+install_dependencies_task:
+  <<: *YARN_CACHE_TEMPLATE
+  timeout_in: 30m
+  eks_container:
+    <<: *BUILD_TEMPLATE
+    cpu: 4
+    memory: 4Gb
+  script:
+    - yarn install --immutable
+
+build_number_task:
+  eks_container:
+    <<: *BUILD_TEMPLATE
+    cpu: 1
+    memory: 2Gb
+  script:
+    - .cirrus/build-number.sh
```

Comment:
If I understand correctly the build number ends up being `version.build_number` ?

Reply:
There are a few "versions". we will use the one inside the `package.json` and then append the build_number for each build

and call it `Project_Version` (see below) But these aren't actually used yet

https://cirrus-ci.com/task/4769891849338880?logs=main#L50